### PR TITLE
Create upgrade-monitoring-services.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-monitoring-services.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+### 第 4 步：使用新的组件包升级 Prometheus 
+
+执行以下命令升级 Prometheus。
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster patch <cluster-name> prometheus-v8.0.0-linux-amd64.tar.gz -R prometheus
+```
+升级完成后，可以打开 Prometheus 主页（地址通常是 http://<Prometheus-server-host-name>:9090）,点击顶部导航菜单“Status” 然后打开 “Runtime & Build Information” 页面，查看 Prometheus 的版本信息，确认升级成功。
```
Comment (dveeden) [inline upgrade-monitoring-services.md:58]:
```suggestion

升级完成后，可以打开 Prometheus 主页（地址通常是 http://<Prometheus-server-host-name>:9090）,点击顶部导航菜单“Status” 然后打开 “Runtime & Build Information” 页面，查看 Prometheus 的版本信息，确认升级成功。
```

File: upgrade-monitoring-services.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+### 第 4 步：使用新的组件包升级 Grafana 
+
+执行以下命令升级 Grafana。
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster patch <cluster-name> grafana-v8.0.0-linux-amd64.tar.gz -R grafana
+```
+升级完成后，可以打开 Grafana 主页（地址通常是 http://<Grafana-server-host-name>:3000, 查看 Grafana 的版本信息，确认升级成功。
```
Comment (dveeden) [inline upgrade-monitoring-services.md:100]:
```suggestion

升级完成后，可以打开 Grafana 主页（地址通常是 http://<Grafana-server-host-name>:3000, 查看 Grafana 的版本信息，确认升级成功。
```

**Review by dveeden — COMMENTED** [review]
Fro the docs-cn-verify-pipeline:
```
[2023-07-14T03:49:42.242Z] upgrade-monitoring-services.md: this file has manual line breaks in the following lines:
[2023-07-14T03:49:42.242Z] 
[2023-07-14T03:49:42.242Z] MANUAL LINE BREAKS: L58
[2023-07-14T03:49:42.242Z] MANUAL LINE BREAKS: L100
```

File: upgrade-monitoring-services.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+TiDB 安装包中自带 Prometheus 组件包，该组件包中的 Prometheus 软件版本是固定的。推荐使用 TiDB 官方安装包中自带的 Prometheus 版本，如果您需要使用更高版本的 Prometheus 软件，可以在Prometheus 官网的[Release Note 页面](https://github.com/prometheus/prometheus/releases)查看新版本特性，选择适合您生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。升级操作需要先从 Prometheus 官网下载所需版本的软件安装包，然后重新构造可被 TiUP 使用的 Prometheus 组件安装包。
+
+### 第 1 步：从 Prometheus 官网下载新版本安装包
+
+从[ Prometheus 官网下载页面](https://prometheus.io/download/)下载组件安装包, 并解压。
+
+
+### 第 2 步：下载 TiDB 官方 Prometheus 安装包
+
+从 [ TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 TiDB-community-server 软件包，解压后找到 Prometheus 安装包，文件名为 “prometheus-v{version}-linux-amd64.tar.gz”。解压该安装包。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:29]:
```suggestion
1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct) 下载 `TiDB-community-server` 软件包，并解压。
2. 在解压文件中，找到 `prometheus-v{version}-linux-amd64.tar.gz`，并解压。
```

File: upgrade-monitoring-services.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:30]: `TiDB-community-server` 软件包的版本是根据用户自己的所使用的 TiDB 版本来下载吗，还是下载最新的呢

File: upgrade-monitoring-services.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+```bash
+tar -xzf prometheus-v{version}-linux-amd64.tar.gz
+```
+
+### 第 3 步：构造新的适用于 TiUP 的 Prometheus 组件包
+
+复制第 1 步中解压的文件，替换第 2 步解压后 prometheus 子目录(./prometheus-v{version}-linux-amd64/prometheus)下的对应文件。替换完成后重新压缩,并将新压缩包命名为“prometheus-v{newversion}.tar.gz”。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:37]:
```suggestion
1. 复制第 1 步中解压的文件，替换第 2 步解压后的 prometheus 子目录 (./prometheus-v{version}-linux-amd64/prometheus) 下的对应文件。
2. 重新压缩替换后的文件，并将新的压缩包命名为 `prometheus-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```

File: upgrade-monitoring-services.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+{{< copyable "shell-regular" >}}
+
+```bash
+cd prometheus-v{version}-linux-amd64.tar.gz
+tar -zcvf ../prometheus-v{new-version}.tar.gz ./
+```
+
+> **注意：**
+>
+> - {new-version} 可以由用户自行指定，无特殊要求。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:48]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+执行以下命令升级 Prometheus。
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster patch <cluster-name> prometheus-{new-version}.tar.gz -R prometheus
+```
+
+升级完成后，可以打开 Prometheus 主页（地址通常是 http://<Prometheus-server-host-name>:9090）,点击顶部导航菜单“Status” 然后打开 “Runtime & Build Information” 页面，查看 Prometheus 的版本信息，确认升级成功。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:60]:
```suggestion
升级完成后，可以打开 Prometheus 主页（地址通常是 `http://<Prometheus-server-host-name>:9090`），点击顶部导航菜单中的 **Status**，然后打开 **Runtime & Build Information** 页面，查看 Prometheus 的版本信息，确认升级成功。
```

**Comment by dveeden** [comment]
Is the procedure for node_exporter the same or is that a different procedure?

File: upgrade-monitoring-services.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+```bash
+tiup cluster patch <cluster-name> grafana-v{new-version}.tar.gz -R grafana
+```
+
+升级完成后，可以打开 Grafana 主页（地址通常是 http://<Grafana-server-host-name>:3000, 查看 Grafana 的版本信息，确认升级成功。
+
+## 升级 AlertManager
+
+TiDB 安装包中直接使用了 AlertManager 官方组件包，升级时只需要下载所需的新版本官方组件包。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:108]:
```suggestion
TiDB 安装包中直接使用了 AlertManager 官方组件包，因此升级 AlertManager 时你只需要下载并安装新版本的官方组件包。
```

File: upgrade-monitoring-services.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+### 第 1 步：从 AlertManager 官网的下载新版本安装包
```
Comment (qiancai) [inline upgrade-monitoring-services.md:110]:
```suggestion
### 第 1 步：从 Prometheus 官网下载新版本安装包
```

File: upgrade-monitoring-services.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+从[ AlertManager 官网下载页面](https://prometheus.io/download/)下载组件安装包。
+
+### 第 2 步：使用新的组件包升级 AlertManager
+
+执行以下命令升级 AlertManager。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:116]:
```suggestion
执行以下命令升级 AlertManager：
```

File: upgrade-monitoring-services.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+{{< copyable "shell-regular" >}}
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:119]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+> **注意：**
+>
+> - {new-version} 可以由用户自行指定，无特殊要求。
+
+### 第 4 步：使用新的组件包升级 Grafana 
+
+执行以下命令升级 Grafana。
+
+{{< copyable "shell-regular" >}}
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:99]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+```bash
+tar -xzf grafana-v{version}-linux-amd64.tar.gz
+```
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+复制第 1 步中解压的文件，替换第 2 步解压后目录(./grafana-v{version}-linux-amd64/prometheus)下的对应文件。替换完成后重新压缩,并将新压缩包命名为“grafna-v{newversion}.tar.gz”。
+
+{{< copyable "shell-regular" >}}
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:84]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+```bash
+tiup cluster patch <cluster-name> prometheus-{new-version}.tar.gz -R prometheus
+```
+
+升级完成后，可以打开 Prometheus 主页（地址通常是 http://<Prometheus-server-host-name>:9090）,点击顶部导航菜单“Status” 然后打开 “Runtime & Build Information” 页面，查看 Prometheus 的版本信息，确认升级成功。
+
+## 升级 Grafana
+
+TiDB 安装包中自带 Grafana 组件包，该组件包中的 Grafana 软件版本是固定的。推荐使用 TiDB 官方安装包中自带的 Grafana 版本，如果您需要使用更高版本的 Grafana 软件，可以在 Grafana 官网的[Release Note 页面](https://grafana.com/docs/grafana/latest/whatsnew/)查看新版本特性，选择适合您生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。升级操作需要先从 Grafana 官网下载所需版本的软件安装包，然后重新构造可被 TiUP 使用的 Grafana 组件安装包。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:64]:
```suggestion
为了更好地兼容 TiDB，推荐使用 TiDB 官方安装包中自带的 Grafana 组件安装包，该组件包中的 Grafana 版本是固定的。如果你需要使用更高版本的 Grafana，可以在 Grafana 官网的 [Release Note 页面](https://grafana.com/docs/grafana/latest/whatsnew/)查看新版本特性，选择适合你生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。

在以下升级步骤中，你需要先从 Grafana 官网下载所需版本的软件安装包，然后将其构造为可被 TiUP 使用的 Grafana 组件安装包。
```

File: upgrade-monitoring-services.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+> **注意：**
+>
+> - 如果监控组件不是由 TiUP 部署和管理，可以直接升级监控组件，无需参考本文档。
+> - TiDB 并未对监控组件新版本的兼容性进行测试，可能存在升级后部分功能无法正常使用的问题，如果遇到问题可以在 TiDB github 中上报 Issue，我们会尽快解决。
+> - 本文所述功能在 TiUP v1.9.0 及后续版本支持，使用本功能前请检查 TiUP 版本号。
+
+## 升级 Prometheus 
+
+TiDB 安装包中自带 Prometheus 组件包，该组件包中的 Prometheus 软件版本是固定的。推荐使用 TiDB 官方安装包中自带的 Prometheus 版本，如果您需要使用更高版本的 Prometheus 软件，可以在Prometheus 官网的[Release Note 页面](https://github.com/prometheus/prometheus/releases)查看新版本特性，选择适合您生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。升级操作需要先从 Prometheus 官网下载所需版本的软件安装包，然后重新构造可被 TiUP 使用的 Prometheus 组件安装包。
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:21]: “该组件包中的 Prometheus 版本是固定的” 这句话可以删除吗，其实也不是完全固定的，有时候还是会升级哇。

File: upgrade-monitoring-services.md [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+升级完成后，可以打开 Prometheus 主页（地址通常是 http://<Prometheus-server-host-name>:9090）,点击顶部导航菜单“Status” 然后打开 “Runtime & Build Information” 页面，查看 Prometheus 的版本信息，确认升级成功。
+
+## 升级 Grafana
+
+TiDB 安装包中自带 Grafana 组件包，该组件包中的 Grafana 软件版本是固定的。推荐使用 TiDB 官方安装包中自带的 Grafana 版本，如果您需要使用更高版本的 Grafana 软件，可以在 Grafana 官网的[Release Note 页面](https://grafana.com/docs/grafana/latest/whatsnew/)查看新版本特性，选择适合您生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。升级操作需要先从 Grafana 官网下载所需版本的软件安装包，然后重新构造可被 TiUP 使用的 Grafana 组件安装包。
+
+### 第 1 步：从 Grafana 官网的下载新版本安装包
+
+从[ Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包, 并解压。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:68]:
```suggestion
从 [Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包，并解压。
```

File: upgrade-monitoring-services.md [hunk 17]
```diff
@@ -0,0 +1,123 @@
...
+
+
+### 第 2 步：下载 TiDB 官方 Grafana 安装包
+
+从 [ TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 TiDB-community-server 软件包，解压后找到 Grafana 安装包，文件名为 “grafana-v{version}-linux-amd64.tar.gz”。解压该安装包。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:73]:
```suggestion
1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 `TiDB-community-server` 软件包，并解压。
2. 在解压文件中，找到 `grafana-v{version}-linux-amd64.tar.gz`，并解压。
```

File: upgrade-monitoring-services.md [hunk 18]
```diff
@@ -0,0 +1,123 @@
...
+
+```bash
+tar -xzf grafana-v{version}-linux-amd64.tar.gz
+```
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+复制第 1 步中解压的文件，替换第 2 步解压后目录(./grafana-v{version}-linux-amd64/prometheus)下的对应文件。替换完成后重新压缩,并将新压缩包命名为“grafna-v{newversion}.tar.gz”。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:81]:
```suggestion
1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/prometheus` 目录下的对应文件。
2. 重新压缩替换后的文件，并将新的压缩包命名为 `grafana-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```

File: upgrade-monitoring-services.md [hunk 19]
```diff
@@ -0,0 +1,123 @@
...
+从 [ TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 TiDB-community-server 软件包，解压后找到 Prometheus 安装包，文件名为 “prometheus-v{version}-linux-amd64.tar.gz”。解压该安装包。
+
+```bash
+tar -xzf prometheus-v{version}-linux-amd64.tar.gz
+```
+
+### 第 3 步：构造新的适用于 TiUP 的 Prometheus 组件包
+
+复制第 1 步中解压的文件，替换第 2 步解压后 prometheus 子目录(./prometheus-v{version}-linux-amd64/prometheus)下的对应文件。替换完成后重新压缩,并将新压缩包命名为“prometheus-v{newversion}.tar.gz”。
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:38]: 重新压缩时候，是压缩 ./prometheus-v{version}-linux-amd64/ 目录吗

File: upgrade-monitoring-services.md [hunk 20]
```diff
@@ -0,0 +1,123 @@
...
+{{< copyable "shell-regular" >}}
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:40]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 21]
```diff
@@ -0,0 +1,123 @@
...
+```bash
+cd grafana-v{version}-linux-amd64.tar.gz
+tar -zcvf ../grafana-v{new-version}.tar.gz ./
+```
+
+> **注意：**
+>
+> - {new-version} 可以由用户自行指定，无特殊要求。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:92]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 22]
```diff
@@ -0,0 +1,123 @@
...
+
+### 第 4 步：使用新的组件包升级 Prometheus 
+
+执行以下命令升级 Prometheus。
+
+{{< copyable "shell-regular" >}}
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:55]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 23]
```diff
@@ -0,0 +1,123 @@
...
+执行以下命令升级 Grafana。
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster patch <cluster-name> grafana-v{new-version}.tar.gz -R grafana
+```
+
+升级完成后，可以打开 Grafana 主页（地址通常是 http://<Grafana-server-host-name>:3000, 查看 Grafana 的版本信息，确认升级成功。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:104]:
```suggestion
升级完成后，可以打开 Grafana 主页（地址通常是 `http://<Grafana-server-host-name>:3000`），查看 Grafana 的版本信息，确认升级成功。
```

File: upgrade-monitoring-services.md (hunk 20 above)
Comment (qiancai) [reply upgrade-monitoring-services.md:40]: 目前文档站会给所有的代码块自动添加复制按钮，无需手动写这个复制标记

**Review by kaaaaaaang — APPROVED** [review]

File: upgrade-monitoring-services.md (hunk 4 above)
Comment (qqqdan) [reply upgrade-monitoring-services.md:30]: 都可以，只是要一个包而已。所以这里没有特别要求。

File: upgrade-monitoring-services.md (hunk 19 above)
Comment (qqqdan) [reply upgrade-monitoring-services.md]: 是的

File: upgrade-monitoring-services.md [hunk 24]
```diff
@@ -0,0 +1,111 @@
...
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/prometheus` 目录下的对应文件。
+2. 重新压缩替换后的文件，并将新的压缩包命名为 `grafana-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
+
+```bash
+cd grafana-v{version}-linux-amd64.tar.gz
+tar -zcvf ../grafana-v{new-version}.tar.gz ./
+```
```
Comment (qiancai) [inline upgrade-monitoring-services.md:84]:
````suggestion
    ```bash
    cd grafana-v{version}-linux-amd64.tar.gz
    tar -zcvf ../grafana-v{new-version}.tar.gz ./
    ```
````

File: upgrade-monitoring-services.md [hunk 25]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### 第 2 步：下载 TiDB 官方 Grafana 安装包
+
+1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 `TiDB-community-server` 软件包，并解压。
+2. 在解压文件中，找到 `grafana-v{version}-linux-amd64.tar.gz`，并解压。
+
+```bash
+tar -xzf grafana-v{version}-linux-amd64.tar.gz
+```
```
Comment (qiancai) [inline upgrade-monitoring-services.md:74]:
````suggestion
    ```bash
    tar -xzf grafana-v{version}-linux-amd64.tar.gz
    ```
````

File: upgrade-monitoring-services.md [hunk 26]
```diff
@@ -0,0 +1,111 @@
...
+
+### 第 3 步：构造新的适用于 TiUP 的 Prometheus 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./prometheus-v{version}-linux-amd64/prometheus` 目录下的对应文件。
+2. 重新压缩替换后的文件，并将新的压缩包命名为 `prometheus-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
+
+```bash
+cd prometheus-v{version}-linux-amd64.tar.gz
+tar -zcvf ../prometheus-v{new-version}.tar.gz ./
+```
```
Comment (qiancai) [inline upgrade-monitoring-services.md:44]:
````suggestion
    ```bash
    cd prometheus-v{version}-linux-amd64.tar.gz
    tar -zcvf ../prometheus-v{new-version}.tar.gz ./
    ```
````

File: upgrade-monitoring-services.md [hunk 27]
```diff
@@ -0,0 +1,111 @@
...
+从 [Prometheus 官网下载页面](https://prometheus.io/download/)下载组件安装包，并解压。
+
+### 第 2 步：下载 TiDB 官方 Prometheus 安装包
+
+1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 `TiDB-community-server` 软件包，并解压。
+2. 在解压文件中，找到 `prometheus-v{version}-linux-amd64.tar.gz`，并解压。
+
+```bash
+tar -xzf prometheus-v{version}-linux-amd64.tar.gz
+```
```
Comment (qiancai) [inline upgrade-monitoring-services.md:34]:
````suggestion
    ```bash
    tar -xzf prometheus-v{version}-linux-amd64.tar.gz
    ```
````

File: upgrade-monitoring-services.md [hunk 28]
```diff
@@ -0,0 +1,111 @@
...
+
+### 第 3 步：构造新的适用于 TiUP 的 Prometheus 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./prometheus-v{version}-linux-amd64/prometheus` 目录下的对应文件。
+2. 重新压缩替换后的文件，并将新的压缩包命名为 `prometheus-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:39]:
```suggestion
2. 重新压缩替换文件后的 `./prometheus-v{version}-linux-amd64` 目录，并将新的压缩包命名为 `prometheus-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```

File: upgrade-monitoring-services.md [hunk 29]
```diff
@@ -0,0 +1,111 @@
...
+2. 在解压文件中，找到 `grafana-v{version}-linux-amd64.tar.gz`，并解压。
+
+```bash
+tar -xzf grafana-v{version}-linux-amd64.tar.gz
+```
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/prometheus` 目录下的对应文件。
+2. 重新压缩替换后的文件，并将新的压缩包命名为 `grafana-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:79]:
```suggestion
2. 重新压缩替换文件后的 `./grafana-v{version}-linux-amd64` 目录，并将新的压缩包命名为 `grafana-v{new-version}.tar.gz`。其中，`{new-version}` 可以由你自行指定。
```

File: upgrade-monitoring-services.md (hunk 15 above)
Comment (qqqdan) [reply upgrade-monitoring-services.md:20]: 对于每个固定的 tidb 版本，里面的 prome 版本是固定的

File: upgrade-monitoring-services.md [hunk 30]
```diff
@@ -0,0 +1,111 @@
+---
+title: 升级集群监控组件
```
Comment (qiancai) [inline upgrade-monitoring-services.md:2]:
```suggestion
title: 升级集群监控组件
summary: 介绍如何升级 TiDB 集群监控组件。
```

File: upgrade-monitoring-services.md [hunk 31]
```diff
@@ -0,0 +1,112 @@
...
+summary: 介绍如何升级 TiDB 集群监控组件 Prometheus、Grafana 和 Alertmanager。
+---
+
+# 升级 TiDB 集群监控组件
+
+使用 TiUP 部署 TiDB 集群时，TiUP 会同时自动部署 Prometheus、Grafana 和 Alertmanager 等监控组件，并且在集群扩容中自动为新增节点添加监控配置。通过 TiUP 自动部署的监控组件并不是这些三方组件的最新版本，如果你需要使用最新的三方组件，可以按照本文的方法升级所需的监控组件。
+
+需要注意的是，TiUP 会使用自己的配置参数覆盖监控组件的配置。如果你直接通过替换监控组件配置文件的方式升级监控组件，在之后对集群进行 `deploy`/`scale-out`/`scale-in`/`reload` 等操作时，该升级可能被 TiUP 所覆盖，导致升级出错。如果需要升级 Prometheus、Grafana 和 Alertmanager，请参考本文的升级步骤。
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:10]:
```suggestion
需要注意的是，TiUP 会使用自己的配置参数覆盖监控组件的配置。如果你直接通过替换监控组件配置文件的方式升级监控组件，在之后对集群进行 `deploy`、`scale-out`、`scale-in`、`reload` 等操作时，该升级可能被 TiUP 所覆盖，导致升级出错。如果需要升级 Prometheus、Grafana 和 Alertmanager，请使用本文介绍的升级步骤。
```

File: upgrade-monitoring-services.md [hunk 32]
```diff
@@ -0,0 +1,112 @@
...
+## 升级 Grafana
+
+为了更好地兼容 TiDB，推荐使用 TiDB 官方安装包中自带的 Grafana 组件安装包，该组件包中的 Grafana 版本是固定的。如果你需要使用更高版本的 Grafana，可以在 Grafana 官网的 [Release Note 页面](https://grafana.com/docs/grafana/latest/whatsnew/)查看新版本特性，选择适合你生产环境的版本，或者咨询 PingCAP 技术支持服务寻求版本建议。
+
+在以下升级步骤中，你需要先从 Grafana 官网下载所需版本的软件安装包，然后将其构造为可被 TiUP 使用的 Grafana 组件安装包。
+
+### 第 1 步：从 Grafana 官网的下载新版本安装包
+
+从 [Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包，并解压。
+
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:66]:
```suggestion
```

File: upgrade-monitoring-services.md [hunk 33]
```diff
@@ -0,0 +1,112 @@
...
+1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 `TiDB-community-server` 软件包，并解压。
+2. 在解压文件中，找到 `grafana-v{version}-linux-amd64.tar.gz`，并解压。
+
+    ```bash
+    tar -xzf grafana-v{version}-linux-amd64.tar.gz
+    ```
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/prometheus` 目录下的对应文件。
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:79]: `./grafana-v{version}-linux-amd64/prometheus` 需要改成  `./grafana-v{version}-linux-amd64/grafana` 吗？

File: upgrade-monitoring-services.md [hunk 34]
```diff
@@ -0,0 +1,112 @@
...
+
+TiDB 安装包中直接使用了 AlertManager 官方组件包，因此升级 AlertManager 时你只需要下载并安装新版本的官方组件包。
+
+### 第 1 步：从 Prometheus 官网下载新版本安装包
+
+从 [Prometheus 官网下载页面](https://prometheus.io/download/#alertmanager)下载 `alertmanager` 组件安装包。
+
+### 第 2 步：使用新的组件包升级 AlertManager
+
+执行以下命令升级 AlertManager：
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:107]:
```suggestion
### 第 2 步：使用新的组件包升级 Alertmanager

执行以下命令升级 Alertmanager：
```

File: upgrade-monitoring-services.md [hunk 35]
```diff
@@ -0,0 +1,111 @@
...
+
+# 升级 TiDB 集群监控组件
+
+使用 TiUP 部署 TiDB 集群时，TiUP 会同时自动部署 Prometheus、Grafana 和 Alertmanager 等监控组件，并且在集群扩容中自动为新增节点添加监控配置。通过 TiUP 自动部署的监控组件并不是这些三方组件的最新版本，如果你需要使用最新的三方组件，可以按照本文的方法升级所需的监控组件。
+
+需要注意的是，TiUP 会使用自己的配置参数覆盖监控组件的配置。如果你直接通过替换监控组件配置文件的方式升级监控组件，在之后对集群进行 `deploy`、`scale-out`、`scale-in`、`reload` 等操作时，该升级可能被 TiUP 所覆盖，导致升级出错。如果需要升级 Prometheus、Grafana 和 Alertmanager，请使用本文介绍的升级步骤。
+
+> **注意：**
+>
+> - 如果监控组件不是由 TiUP 部署和管理，可以直接升级监控组件，无需参考本文档。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:14]: “可以直接升级监控组件”是指“可以直接通过替换监控组件配置文件的方式升级监控组件”吗

**Review by qiancai — APPROVED** [review]

File: upgrade-monitoring-services.md [hunk 36]
```diff
@@ -0,0 +1,111 @@
...
+从 [Prometheus 官网下载页面](https://prometheus.io/download/#alertmanager)下载 `alertmanager` 组件安装包。
+
+### 第 2 步：使用新的组件包升级 Alertmanager
+
+执行以下命令升级 Alertmanager：
+
+```bash
+tiup cluster patch <cluster-name> alertmanager-v{new-version}-linux-amd64.tar.gz -R alertmanager
+```
+
```
Comment (qiancai) [inline upgrade-monitoring-services.md:111]:
```suggestion
```

**Review by qiancai — APPROVED** [review]

File: upgrade-monitoring-services.md (hunk 33 above)
Comment (qiancai) [reply upgrade-monitoring-services.md:78]: @qqqdan PTAL

File: upgrade-monitoring-services.md [hunk 37]
```diff
@@ -0,0 +1,110 @@
...
+1. 在 [TiDB 官网下载页面](https://cn.pingcap.com/product/#SelectProduct)下载 `TiDB-community-server` 软件包，并解压。
+2. 在解压文件中，找到 `grafana-v{version}-linux-amd64.tar.gz`，并解压。
+
+    ```bash
+    tar -xzf grafana-v{version}-linux-amd64.tar.gz
+    ```
+
+### 第 3 步：构造新的适用于 TiUP 的 Grafana 组件包
+
+1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/prometheus` 目录下的对应文件。
```
Comment (qqqdan) [inline upgrade-monitoring-services.md:78]:
```suggestion
1. 复制第 1 步中解压的文件，替换第 2 步解压后的 `./grafana-v{version}-linux-amd64/` 目录下的对应文件。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**